src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -139,7 +139,7 @@ def frequency(self) -> GridFrequency:
 
         return self._frequency_instance
 
-    def voltage(self) -> VoltageStreamer:
+    def voltage_per_phase(self) -> VoltageStreamer:
         """Return the 3-phase voltage measuring point."""
```

Comment:
Maybe `Return the per-phase voltage measuring point.`?

Reply:
true, fixed.